### PR TITLE
#310: improved documentation

### PR DESCRIPTION
**Pull request by AnastasiiaSergienko**

### CODE REVIEW

File: doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md [hunk 1]
```diff
@@ -184,7 +182,7 @@ And we also need two corresponding test classes:
     }
     ```
 
-10. Now repeat the previous two steps for all **other kinds of capabilities**.
+9. Now repeat the previous two steps for all **other kinds of capabilities**.
```
Comment (redcatbear) [inline doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md]:
```suggestion
9. Repeat the previous two steps for all **other kinds of capabilities**.
```

File: doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md [hunk 2]
```diff
@@ -89,9 +89,7 @@ And we also need two corresponding test classes:
    }
    ```
   
-6. **Add the dialect name** (here `ATHENA`) **to the list of dialects**  for which the `JbdcAdapterFactory` is responsible in the method `getSupportedAdapterNames()`.
-
-7. Create a **new unit test class** for the dialect in the test package you created before: `com.exasol.adapter.dialects.athena.AthenaSqlDialectTest` that tests class `AthenaSqlDialect`.
+6. Create a **new unit test class** for the dialect in the test package you created before: `com.exasol.adapter.dialects.athena.AthenaSqlDialectTest` that tests class `AthenaSqlDialect`.
```
Comment (redcatbear) [inline doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md]: Use `1.` for numbers to make your life easier.

File: doc/development/developing-sql-dialect/step_by_step_guide_to_writing_your_own_dialect.md [hunk 3]
```diff
@@ -9,6 +9,20 @@ In the following section we will walk through the process of developing such a S
 _Athena is based on the Open Source project [Apache Presto](https://prestodb.github.io) which in the own words of the Presto team is a "distributed SQL query engine for Big Data". 
 In short it's a cluster of machines digging through large amounts of data stored on a distributed file system._
 
+## Prerequisites
+
+To start developing Virtual Schemas you should know:
+
+* How to use [git](https://git-scm.com/)
+* How to use [Maven](https://maven.apache.org/). Check the list of dependencies and plugins that we use in the project at the end of [README](../../../README.md) file.
+* How to setup Maven project in an IDE of your preference.
```
Comment (redcatbear) [inline doc/development/developing-sql-dialect/step_by_step_guide_to_writing_your_own_dialect.md]:
```suggestion
* How to setup a Maven project in the IDE of choice. 
```

File: doc/user-guide/user_guide.md [hunk 4]
```diff
@@ -5,10 +5,12 @@ The purpose of this page is to provide detailed instructions for each of the sup
 * How does the **CREATE VIRTUAL SCHEMA** statement look like, i.e. which properties are required.
 * **Data source specific notes**, like authentication with Kerberos, supported capabilities or things to consider regarding the data type mapping.
 
-## Before you Start
+## Prerequisites
 
-Please note that the syntax for creating adapter scripts is not recognized by all SQL clients. 
-[DBeaver](https://dbeaver.io/) for example. If you encounter such a problem, try a different client.
+To start using Virtual Schemas you should know:
```
Comment (redcatbear) [inline doc/user-guide/user_guide.md]:
```suggestion
Before you can start using Virtual Schemas you should know:
```

File: doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md (hunk 2 above)
Comment (AnastasiiaSergienko) [reply doc/development/developing-sql-dialect/implementing_mandatory_sql_dialect_classes.md]: fixed

**Pull request merged**